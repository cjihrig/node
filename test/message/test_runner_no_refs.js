// Flags: --no-warnings
'use strict';
require('../common');
const test = require('test_runner');

// When run alone, the test() below does not keep the event loop alive. This
// test verifies that the test harness keeps the event loop alive anyway.
test('does not keep event loop alive pass', async (t) => {
  await t.test('+does not keep event loop alive pass', async (t) => {
    return new Promise((resolve) => {
      setTimeout(resolve, 1000).unref();
    });
  });
});
