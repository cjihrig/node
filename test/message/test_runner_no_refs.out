TAP version 13
    ok 1 - +does not keep event loop alive pass
      ---
      duration_ms: *
      ...
    1..1
ok 1 - does not keep event loop alive pass
  ---
  duration_ms: *
  ...
1..1
# tests 1
# pass 1
# fail 0
# skipped 0
# todo 0
# duration_ms *
